nitransforms/linear.py

```diff
@@ -326,6 +342,13 @@ def load(filename, fmt='X5', reference=None):
     # elif fmt.lower() == 'afni':
     #     parameters = LPS.dot(self.matrix.dot(LPS))
     #     parameters = parameters[:3, :].reshape(-1).tolist()
+    elif fmt.lower() in ('fs', 'lta'):
+        with open(filename) as ltafile:
+            lta = LinearTransformArray.from_fileobj(ltafile)
+        assert lta['nxforms'] == 1  # ever have multiple transforms?
+        if lta['type'] != 1:
+            lta.as_type(1)
+        matrix = lta['xforms'][0]['m_L']
```

Comment:
matrix is of size N x (D + 1) x (D + 1), where N is the number of transforms and D the dimension (i.e., D 
belongs to {2, 3})